From: Michael Olbrich <m.olbrich@pengutronix.de>
Subject: add missing #include <linux/types.h>

Without "#include <linux/types.h>" __u32 is undefined in linux/hiddev.h on PPC

Signed-off-by: Michael Olbrich <m.olbrich@pengutronix.de>

---
 extras/hid2hci/hid2hci.c |    1 +
 1 file changed, 1 insertion(+)

Index: b/extras/hid2hci/hid2hci.c
===================================================================
--- a/extras/hid2hci/hid2hci.c
+++ b/extras/hid2hci/hid2hci.c
@@ -28,6 +28,7 @@
 #include <string.h>
 #include <getopt.h>
 #include <sys/ioctl.h>
+#include <linux/types.h>
 #include <linux/hiddev.h>
 #include <usb.h>
 
